charts/common/templates/_annotations.tpl

```diff
@@ -0,0 +1,10 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Kubernetes github annotations
+*/}}
+{{- define "common.annotations.github" -}}
+github.com/branch: {{ .Values.github.prbranch }}
+github.com/pull-request: {{ .Values.github.prurl }}
+github.com/pull-request-owner: {{ .Values.github.prowner }}
+github.com/repository: {{ .Values.github.repourl }}
```

Comment:
Can we use the following for the API?

```
fluidtruck:
  app: ""
  team: ""
  env: ""

github:
  repo: ""
  branch: ""
  pr:
    number: ""
    owner: ""
```

This would then be used with:

```
--set fluidtruck.app="go-demo" \
--set fluidtruck.team="devops" \
--set github.repo="fluidtruck/go-demo" \
--set github.branch="feat-new-work" \
--set github.pr.number="24" \
--set github.pr.owner="ellisio"
```

You can then generate the following annotations:

```
github.com/repository: {{ .Values.github.repo }}
github.com/branch: {{ .Values.github.branch }}
{{- if .Values.pr.number -}}
github.com/pull-request: {{ .Values.pr.number }}
github.com/pull-request-owner: {{ .Values.pr.owner }}
github.com/pull-request-url: https://github.com/{{ .Values.github.repo }}/pull/{{ .Values.pr.number }}
{{- end -}}
```

Reply:
Ok, so, with these changes, the idea is we should be able to...

```
fluidtruck:
  app:
  team:
  env:

github:
  repo:
    name:
    url:
  pr:
    branch:
    number:
    owner:
    url:
  release:
    version:
    url:
```


All:
```
Labels:
--set fluidtruck.app="go-demo" \
--set fluidtruck.team="devops" \
--set fluidtruck.env="" \
--set github.repo.name="go-demo" \

Annotations:
--set github.repo.url="https://github.com/fluidtruck/go-demo" \

```

PR:
```
Labels:
--set github.pr.branch="feat-new-work" \
--set github.pr.number="24" \

Annotations:
--set github.pr.owner="ellisio" \
--set github.pr.url="https://github.com/fluidtruck/go-demo/pull/71" \
```

Release:
```
Labels:
--set github.release.version="0.5.0" \

Annotations:
--set github.release.url="https://github.com/fluidtruck/go-demo/releases/tag/v0.5.0" \
```